Gemfile.lock

```diff
@@ -104,7 +104,7 @@ GEM
       method_source (~> 0.9.0)
     public_suffix (4.0.1)
     puma (3.12.2)
-    rack (2.0.7)
+    rack (2.0.8)
```

Comment:
what does this do?

Reply:
Don't know, I think it happened when I ran `bundle update`